test/fn/create-users.spec.js

```diff
@@ -7,14 +7,16 @@ const api = require('../api-stub');
 const environment = require('../../src/lib/environment');
 
 const createUsers = rewire('../../src/fn/create-users');
+const userPrompt = rewire('../../src/lib/user-prompt');
 
 let readLine;
 const mockTestDir = testDir => sinon.stub(environment, 'pathToProject').get(() => testDir);
 
 describe('create-users', () => {
   beforeEach(() => {
     readLine = { keyInYN: sinon.stub() };
-    createUsers.__set__('readline', readLine);
+    userPrompt.__set__('readline', readLine);
+    createUsers.__set__('userPrompt', userPrompt);
```

Comment:
I think there's a simpler way to achieve this, without doing a double rewire and set
Up top, when you `let readLine`, replace with 
```
const readLine = require('readline-sync');
```

and here just do:
```
sinon.stub(readLine, 'keyInYN');
```

I think this can be applied to most, if not all the tests below.

Reply:
Have updated to your recommendations.